demos/demo_references.bib

```diff
@@ -25,6 +25,14 @@ @InBook{2015:lmscup
                   {http://www1.maths.leeds.ac.uk/~matak/lms-cup2015.pdf}
 }
 
+@Manual{Gidel:2017,
```

Comment:
Pretty sure that's not true.  Various bibtex guides say [for article] "Required fields: author, title, journal, year. Optional fields: volume, number, pages, month, note."